examples/tabs/index.html

```diff
@@ -1,582 +1,402 @@
-            &lt;http://bbc.co.uk/&gt;  &lt;http://cbc.ca/&gt;  &lt;http://dbp.de/&gt;  &lt;https://webfoundation.org/&gt;
-            &lt;http://www.zenith.org/&gt; ) .
-
-</pre>
-          </td>
-
-          <td class='output'></td>
-        </tr>
-
-        <tr>
-          <td class='input'>
-            <pre>
-           #  Reverse the list
-           &lt;#tabs&gt; a ui:Tabs;
-             ui:predicate trip:attachmentList;
-             ui:orientation 3; ui:ordered true;
-             ui:subject &lt;#it&gt;.
-           &lt;#it&gt;
-             trip:attachmentList (
-             &lt;http://www.zenith.org/&gt;
-             &lt;https://webfoundation.org/&gt;
-             &lt;http://dbp.de/&gt;
-             &lt;http://cbc.ca/&gt;
-             &lt;http://bbc.co.uk/&gt;
-             &lt;http://aclu.org/&gt;
-             &lt;http://acm.org/&gt;
-              ) .
-
-</pre>
-          </td>
-
-          <td class='output'></td>
-        </tr>
-
-        <tr>
-          <td class='input'>
-            <pre>
-         # Remove all but one
-         &lt;#tabs&gt; a ui:Tabs;
-           ui:predicate trip:attachmentList;
-           ui:orientation 3; ui:ordered true;
-           ui:subject &lt;#it&gt;.
-         &lt;#it&gt;
-           trip:attachmentList ( &lt;https://webfoundation.org/&gt; ) .
 
-</pre>
-          </td>
+<h1>The Tabs module</h1>
+
+<p>
+    On this page you'll find some examples from the Tabs module.
+</p>
+
+<article id="tabWidget-with-collection">
+    <h2>
+        <a href="#tabWidget-with-collection">
+            <code>tabWidget</code>: Populating content using a RDF collection
+        </a>
+    </h2>
+    <p>
+        In this first example we'll see how you can generate a tabs widget using triples that's formatted as a
+        collection
+        in RDF. See <a href="http://127.0.0.1:8080/Documentation/api/modules/_tabs_.html#tabwidget">API
+        documentation</a>
+        for more info about how to use the tabWidget.
+    </p>
+    <script id="script-tabWidget-with-collection">
+      window.addEventListener('DOMContentLoaded', () => {
+        console.log(UI.store)
+        const subject = UI.rdf.namedNode(location.origin + '#subject1')
+        const predicate = UI.rdf.namedNode(location.origin + '#predicate1')
+        const tabs = UI.tabs.tabWidget({
+          subject,
+          predicate,
+          renderMain: (bodyMain, subject) => {
+            bodyMain.innerText = `Content of ${subject.uri}`
+          }
+        })
+        document.querySelector('#demo-tabWidget-with-collection').appendChild(tabs)
+      })
+    </script>
+    <p>
+        The triples belows shows the minimum data you need for to start using the tabWidget in combination with
+        UI.store.
+    </p>
+    <pre id="viewData-tabWidget-with-collection"></pre>
+    <p>
+        Note that you would probably use UI.rdf.namedNode directly from rdflib.js, but we've used the convenience
+        reference
```

Comment:
which convenience reference? Looks like you do use UI.rdf.namedNode

Reply:
`UI.rdf` _is_ the convenience reference. Maybe convenience reference isn't the best word?

Reply:
I'll remove the word convenience reference and be a bit more explicit about what you would do instead.